### PR TITLE
Update container version in Jenkinsfile

### PR DESCRIPTION
**Pull request by ericharper**

### CODE REVIEW

**Review by okuchaiev — APPROVED** [review]
lgtm

File: Dockerfile [hunk 1]
```diff
@@ -89,7 +89,7 @@ COPY . .
 
 # start building the final container
 FROM nemo-deps as nemo
-ARG NEMO_VERSION=1.0.0rc1
+ARG NEMO_VERSION=main
```
Comment (titu1994) [inline Dockerfile:92]: This should revert to 1.0.0rc1

Comment (okuchaiev) [reply Dockerfile]: why though? 1.0.0rc1 was never tested with 21.03 container?

File: Dockerfile [hunk 2]
```diff
@@ -68,7 +77,7 @@ RUN COMMIT_SHA=f546575109111c455354861a0567c8aa794208a2 && \
     rm -rf ../tests test ../tensorflow_binding
 
 # uninstall stuff from base container
-RUN pip uninstall -y sacrebleu
+RUN pip uninstall -y sacrebleu torchtext
```
Comment (titu1994) [inline Dockerfile:80]: Wait, torchtext is being deleted here again, this is unecessary now that we built from source.

**Review by titu1994 — COMMENTED** [review]
LGTM

File: Dockerfile (hunk 2 above)
Comment (ericharper) [reply Dockerfile]: I moved it earlier now.

**Comment by titu1994** [comment]
LGTM

**Pull request merged**